.prettierrc.json

```diff
@@ -1,7 +1,15 @@
 {
-	"trailingComma": "all",
-	"arrowParens": "always",
+	"endOfLine": "lf",
+	"semi": true,
 	"singleQuote": true,
-	"printWidth": 80,
-	"semi": true
+	"tabWidth": 2,
+	"trailingComma": "es5",
+	"arrowParens": "always",
+	"importOrder": [
+		"^react$",
+		"^(?!^react$|^@/|^[./]).*",
```

Comment:
If think this is useless as there is no React in this project.

Reply:
Oh yes, I will remove the entire plugin because it doesn't seems to work with `require`